index.bs

```diff
@@ -2019,6 +2022,8 @@ To <dfn>obtain an event-level report body</dfn> given an [=attribution report=]
     :: |report|'s [=event-level report/trigger data=], [=serialize an integer|serialized=]
     : "`report_id`"
     :: |report|'s [=event-level report/report ID=]
+    : "`scheduled_report_time`"
+    :: |report|'s [=event-level report/original report time=] in seconds since the UNIX epoch, [=serialize an integer|serialized=]
```

Comment:
Is it intentional that we aren't rounding this value to the nearest minute/hour/day? For the [equivalent field in aggregatable report shared info](https://wicg.github.io/attribution-reporting-api/#aggregatable-report-shared-info), we do so.

Reply:
Ah, my mistake.